src/jspm-hmr-server.ts

```diff
@@ -66,7 +66,7 @@ export function createServer(options: ServerOptions): JspmHmrServer {
 
     const proxy = httpProxy.createProxyServer();
     app.use(proxyRoute, (req, res) => {
-      req.url = req.baseUrl;
+      req.url = `${req.baseUrl}${req.url}`;
```

Comment:
@amygdaloideum thanks, looks good to me 👍  
I was planning to write some tests for basic features, I gonna use this as a test case 
good job!